pydoctor/themes/base/apidocs.css

```diff
@@ -269,18 +287,23 @@ ul ul ul ul ul ul ul {
         width: 35%;
     }
 
+    /* Argument name + type column table  */
+    .fieldTable tr td.fieldArgContainer {
+        width: 175px;
+    } 
+
```

Comment:
Is this required ?
```suggestion

```

Th current width is 250px and it's already a bit small...